src/apps/profiles/views.py

```diff
@@ -1,16 +1,22 @@
 # from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth import authenticate, login
 from django.http import HttpResponse
 # from django.shortcuts import render
+from django.shortcuts import render, redirect
+
+from .forms import SignUpForm
 
 
 def sign_up(request):
-    return HttpResponse("Not implemented yet!")
+    if request.method == 'POST':
```

Comment:
There should already be a signup we can link into, I think? but it might be nice to have our own, like to add a "check this box to agree to terms" thing or something..

Reply:
We decided this is good right? Pretty much the default implementation

Reply:
Yeah this seems fine